src/components/common/NameRequestInfo.vue

```diff
@@ -111,7 +111,7 @@ export default class NameRequestInfo extends Vue {
   }
 
   /** The entity title  */
-  protected entityTypeDescription (): string {
```

Comment:
I changed all of these from functions to getters. As well as cleaner, it may be a bit faster (since they're cached, and depending on when they're evaluated).

Anyway, they work fine:

![image](https://user-images.githubusercontent.com/10002889/203180280-521c686d-a5dd-4042-a073-da1d9c5e5d22.png)